tests/conftest.py

```diff
@@ -79,13 +89,26 @@ def _(dbname, template=None):
 
 
 @pytest.fixture(scope="session")
-def setup_db():
+def setup_db(request: pytest.FixtureRequest):
     dbname = "procrastinate_test_template"
     db_create(dbname=dbname)
     connector = testing.InMemoryConnector()
 
-    with db_executor(dbname) as execute:
-        execute(schema.SchemaManager(connector=connector).get_schema())
+    if not request.config.getoption("--skip-post-migration"):
+        with db_executor(dbname) as execute:
+            execute(schema.SchemaManager(connector=connector).get_schema())
+    else:
+        schema_manager = schema.SchemaManager(connector=connector)
+        migrations_path = Path(schema_manager.get_migrations_path())
+        migrations = sorted(migrations_path.glob("*.sql"))
+        last_migration = migrations[-1]
+        if "_post_" in last_migration.name:
+            migrations.remove(last_migration)
```

Comment:
This is not right.
1/ There may be multiple post_migrations
2/ maybe the last release was 3.4, the current release we're working on will be 3.5, and 3.4 had no migrations and the last migrations are the ones for 3.3, and in that case we don't want to test 3.5 without the post-migrations of 3.3. The only post-migrations we want to skip are those that haven't been released yet.

Reply:
Sure. Sorry, I should have made this a draft. It's just temporary, but I think I will fetch the latest tag in the nox session and then provide it as an option to pytest (seems to me more like part of the environment).